packages/sw-build/src/lib/get-file-manifest-entries.js

```diff
@@ -2,7 +2,7 @@ const errors = require('./errors');
 const filterFiles = require('./utils/filter-files');
 const getCompositeDetails = require('./utils/get-composite-details');
 const getFileDetails = require('./utils/get-file-details');
-const getStringDetails = require('./utils/get-string-hash');
+const getStringDetails = require('./utils/get-string-details');
```

Comment:
This was changed from get-string-hash -> get-string-details. I found the original name more specific description wise. Although you're generating hash -> updating -> returning digest. Small nit to consider.